tests/service/ProjectAnalysisTests.fs

```diff
@@ -4325,3 +4325,63 @@ let ``Test project34 should report correct accessibility for System.Data.Listene
         |> Option.get
 
     listenerFuncEntity.Accessibility.IsPrivate |> shouldEqual true
+
+module Project35 = 
+    open System.IO
+
+    let fileName1 = Path.ChangeExtension(Path.GetTempFileName(), ".fs")
+    let base2 = Path.GetTempFileName()
+    let dllName = Path.ChangeExtension(base2, ".dll")
+    let projFileName = Path.ChangeExtension(base2, ".fsproj")
+    let fileSource1 = """
+type Test =
+    let curriedFunction (one:int) (two:float) (three:string) =
```

Comment:
Does  ValReprInfo = None for these declarations today?  From the look of what you have above, I was expecting the test case to be a function defined in an expression:

    let Test() =
         let curriedFunction (one:int) (two:float) (three:string) = ...

not a function defined in a class.